### PR TITLE
Fix logic to build internal/external versions on update_repos management command

### PR DESCRIPTION
**Pull request by saadmk11**
I'm not quite sure why we have the else statement here isn't `slugs` a required argument here?

closes https://github.com/readthedocs/readthedocs.org/issues/6427

### CODE REVIEW

File: readthedocs/core/management/commands/update_repos.py [hunk 1]
```diff
@@ -89,6 +90,26 @@ def handle(self, *args, **options):
...
+                ):
+                    trigger_build(project=version.project, version=version)
+            elif version == EXTERNAL:
+                log.info('Updating all external versions')
+                for version in Version.external.filter(
+                        active=True,
+                        uploaded=False,
+                ):
+                    trigger_build(project=version.project, version=version)
+            elif version and version != 'all':
```
Comment (stsewd) [inline readthedocs/core/management/commands/update_repos.py]:
```suggestion
            elif version:
```

**Comment by saadmk11** [comment]
> If slugs isn't given we build all docs, otherwise only the docs that have that slugs

If the slugs are not given then shouldn't we get an error?

`manage.py update_repos: error: the following arguments are required: slugs`

**Comment by stsewd** [comment]
oh, good catch! That shouldn't be a required argument. Could you fix that here?

**Comment by saadmk11** [comment]
> oh, good catch! That shouldn't be a required argument. Could you fix that here?

Done! :)

**Pull request merged**